packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -455,6 +455,7 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 					await verifyBuildScript(framework, projectPath, logStream);
 					await storeDiff(framework, projectPath);
 				} catch (e) {
+					console.error("ERROR", e);
```

Comment:
@threepointone was this meant to be `console.error` or `logger.error`?

Reply:
oh, totally missed this was in `create-cloudflare`. Is the log relevant here then? I see we're asserting on the fail message...not sure if that needs to be logged for that assertion to pass

Reply:
I added it while debugging, but it's not a bad idea to keep it here, because otherwise you have to see logs, which are hard to do in CI (unless I'm mistaken?)